judo-ui-react/src/main/resources/actor/README.md.hbs

```diff
@@ -75,6 +75,19 @@ pnpm run build:ci
 
 The output of the build is placed under the `dist/` folder.
 
+{{# if isMUILicensed }}
```

Comment:
I believe this information could always be generated, so that even if a project is on a community license, it is still clear which environment variables need to be updated "before" upgrading.